.travis.yml

```diff
@@ -28,6 +28,7 @@ matrix:
     - rust: nightly
 
 script:
+  - cd structopt-derive && cargo build && cd ..
```

Comment:
Why do you think this is needed?

Reply:
To both reproduce the problem and identify anything like it in the future

Reply:
cargo build builds both crates anyway, so it's kind of meaningless 🤷‍♂

Reply:
Except that if structopt includes a feature that structopt-derive needs (as happened here) we won't be able to spot the problem. This build step means we can be sure structopt-derive is capable of compiling correctly on it's own.

Reply:
>  structopt includes a feature that structopt-derive needs

That's not what happened here. `structopt-derive` (not `sturctopt`!) depends on `proc-macro-error`, which used to depend on `rustversion` up until the yesterday's release, which enables the "full" feature. I've ditched `rustversion` yesterday, so the dependency on `proc-macro-error` doe not result in `syn-full` enabled anymore. As you can see, if `structopt-derive` fails to build, structopt would fail too.

`structopt` on it's own is innocent here, this was caused by the derive and derive only.

Reply:
Disregard my last, I was talking about `cargo test`, not `cargo build`. But still, I doubt this problem will ever arise again (yeah, sure).